packages/forma-36-website/src/pages/components/example/index.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: 'Example Component'
+type: 'component'
+github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/FormLabel'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-formlabel--default'
+---
+
+Add a description about the component and its function in the user interface here
+E.g: TextLinks communicate actions and highlight linked resources to users.
+
+Add an example of the component like this:
+
+```jsx
+<React.Fragment>
+  <Button buttonType="primary">Primary</Button>
+</React.Fragment>
+```
+
+## Table of contents
+
+1. [Best practices](#best-practices)
+2. [Variations](#variations)
+   1. [Button types](#button-types)
+3. [Copy considerations](#copy-considerations)
+4. [Accessibility](#accessibility)
+
+## Best practices <a name="best-practices" />
+
+Add best practices for usage in the UI, think about copy and accessibility or general UX guidelines
+E.g (Button Component):
+
+- Position buttons consistently in the interface
+- Reduce complexity by using a small number of actions. Too many actions can create confusion.
+
+## Variations <a name="variations" />
+
+Describe important properties on the component and give examples. Important properties configure a component to serve its main purposes
+
+**_E.g: Button Component_**
+
+The Button component can be configured in a number of different ways. Here is a guide to when to use certain variations.
```

Comment:
The Button component can be configured in a number of different ways. Follow this guidance to determine when to use certain variations.